launcher/ui/instanceview/InstanceView.cpp

```diff
@@ -478,6 +479,38 @@ void InstanceView::paintEvent([[maybe_unused]] QPaintEvent* event)
 #endif
     option.widget = this;
 
+    if (model()->rowCount() == 0) {
+        painter.save();
+        const QString line1 = tr("Welcome!");
+        const QString line2 = tr("Add an instance to get started.");
```

Comment:
```suggestion
        const QString line2 = tr("Click "Add Instance" to get started.");
```

Reply:
should be fixed now .
Also made the text a little smaller.